src/core/evaluator.js

```diff
@@ -1605,11 +1617,8 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
               }
 
               var name = args[0].name;
-              if (xobjsCache.key === name) {
-                if (xobjsCache.texts) {
-                  Util.appendToArray(textContent.items, xobjsCache.texts.items);
-                  Util.extendObj(textContent.styles, xobjsCache.texts.styles);
-                }
+              // Skip if /Form XObject don't have text.
```

Comment:
This comment makes no sense, since what you're caching now is XObjects that does *not* have `/Subtype = /Form` set; let's just remove it.